tests/test_errors.py

```diff
@@ -0,0 +1,21 @@
+# SPDX-License-Identifier: BSD-2-Clause
+import unittest
+
+from chipflow_lib.errors import ChipFlowError
+
+
+class TestErrors(unittest.TestCase):
+    def test_chipflow_error(self):
+        """Test that ChipFlowError can be instantiated and raised"""
+        # Test instantiation
+        error = ChipFlowError("Test error message")
+        self.assertEqual(str(error), "Test error message")
+
+        # Test raising
+        with self.assertRaises(ChipFlowError) as cm:
```

Comment:
several of the tests, like this one, don't really seem to be testing anything and just seem to be adding lines of code to maintain?

tests are good, but only when they actually test functionality that matters and don't just make making changes harder (the previous chipflow cloud stuff was a bad offender here)